### PR TITLE
Add a new "Open in Terminal" command

### PR DESCRIPTION
**Pull request by jsuarezruiz**
Add a new "Open in Terminal" command (like Reveal in Finder)

Screenshot 2019-06-18 at 10 51 21
Screenshot 2019-06-18 at 10 51 34

Feature VSTS #786721

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectPadContextMenu.addin.xml [hunk 1]
```diff
@@ -118,6 +118,7 @@
 	</Condition>
 	<Condition id="ItemType" value="IFolderItem|IFileItem">
 		<CommandItem id = "MonoDevelop.Ide.Commands.FileCommands.OpenContainingFolder" />
+		<CommandItem id = "MonoDevelop.Ide.Commands.FileCommands.OpenInTerminal" />
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectPadContextMenu.addin.xml]: There is already an OpenInTerminal command in the Tools submenu for the folder.

Comment (jsuarezruiz) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectPadContextMenu.addin.xml]: Right. My original idea was to expose in a more visible way the option to open the terminal from the ProjectPad. I also added the option in the document tabs (removed).
After talking with @sevoku, the idea is the following:
- Move the "Open in Terminal" option from the Tools submenu to something more direct.
- Show the option not only with folders, also with files (will use the folder that contains the file).

What do you think?

Comment (sevoku) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectPadContextMenu.addin.xml]: @jsuarezruiz after discussing this with @slluis offline, I agree that we don't really need that for files, this is a folder operation. The main problem is discoverability, I didn't know that we have that at all. However it's due to the context how we use the product, same can be valid for other users with other features, so we need to make a cut at some point. The main problem that I see here is a bit different, for regular folders it's the only option in the tools menu, and submenus don't make sense with just one option, however this is not easy to solve with our command structure and so out of scope here.

Comment (jsuarezruiz) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectPadContextMenu.addin.xml]: Ok. I am going to close the PR.

**Pull request closed without merging**